demos/demo_imi_pq.cpp

```diff
@@ -121,7 +121,7 @@ int main ()
                 elapsed() - t0, nb);
 
         std::vector <float> database (nb * d);
-        std::vector <long> ids (nb);
+        std::vector <int64_t> ids (nb);
```

Comment:
This should technically be `faiss::Index::idx_t`.

Reply:
thins for advice